Formula/ignition-garden.rb

```diff
@@ -27,7 +20,7 @@ class IgnitionGarden < Formula
   depends_on "ignition-msgs8"
   depends_on "ignition-physics5"
   depends_on "ignition-plugin1"
-  depends_on "ignition-rendering6"
+  depends_on "ignition-rendering7"
   depends_on "ignition-sensors6"
```

Comment:
Oops, I think it's missing some dependency bumps

Reply:
yeah, I'm debugging some changes to the bump scripts